docs-web/Makefile

```diff
@@ -0,0 +1,28 @@
+NETLIFY_ROOT	=	/opt/build/repo
+SITE_ROOT			= ${NETLIFY_ROOT}/site
+BUILDDIR			= ${NETLIFY_ROOT}/docs-web/build
+#DOCKER_IMAGE 	=
+
+.PHONY: all build clean setup
+
+#help:
+
+all: clean setup build
+#docker-all: docker setup build
+
+clean:
+	rm -rf ${SITE_ROOT}
+
+setup:
+	mkdir ${SITE_ROOT}
+	@git clone -q --single-branch --branch master https://$(GITHUB_PAT)@github.com/nginxinc/docs-nginx-com.git "${SITE_ROOT}"/docs-nginx-com
+	pip install -r ${SITE_ROOT}/docs-nginx-com/requirements.txt
+	rsync -avOz "${NETLIFY_ROOT}"/docs-web/ ${SITE_ROOT}/docs-nginx-com/source/nginx-ingress-controller/ --exclude ${NETLIFY_ROOT}/docs-web/.netlify --exclude ${NETLIFY_ROOT}/docs-web/Makefile
```

Comment:
`--exclude ${NETLIFY_ROOT}/docs-web/.netlify` - this file doesn't exist in our repo, right?

Reply:
Correct! Removing.